cmd2/rl_utils.py

```diff
@@ -33,6 +33,38 @@ class RlType(Enum):
 if 'pyreadline' in sys.modules:
     rl_type = RlType.PYREADLINE
 
+    ############################################################################################################
+    # pyreadline is incomplete in terms of the Python readline API. Add the missing functions we need.
```

Comment:
It’s a shame that pyreadline isn’t being actively maintained.  If you are feeling motivated, you might want to reach out to the old maintainers and see if they would like some help ...